src/Symfony/Component/PropertyInfo/PropertyDocBlockExtractorInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+use phpDocumentor\Reflection\DocBlock;
+
+/**
+ * Extract a property's doc block.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+interface PropertyDocBlockExtractorInterface
+{
+    /**
+     * Get doc block for a property.
```

Comment:
I would detail the fallback logic.

Reply:
Thank you. 

Sure let's put that in the contract too.